Remotion/Data/DomainObjects.UnitTests/Persistence/Rdbms/RdbmsProviderTest.cs

```diff
@@ -70,11 +70,55 @@ public override void SetUp ()
 
       _provider = new TestableRdbmsProvider(
           TestDomainStorageProviderDefinition,
+          TestDomainStorageProviderDefinition.ConnectionString,
           NullPersistenceExtension.Instance,
           _commandFactoryMock.Object,
           () => _connectionCreatorMock.Object.CreateConnection());
     }
 
+    [Test]
+    public void RdbmsProvider_withConnectionStringContainedInProviderDefinition ()
```

Comment:
here, and the rest of the file, "with" should be "With" (first letter in upper case)